tests/unittest_builder.py

```diff
@@ -614,7 +615,11 @@ def test_module_base_props(self) -> None:
         self.assertEqual(module.package, 0)
         self.assertFalse(module.is_statement)
         self.assertEqual(module.statement(), module)
-        self.assertEqual(module.statement(), module)
+        with pytest.warns(DeprecationWarning) as warnings:
+            module.statement()
+            assert len(warnings) == 1
```

Comment:
```suggestion
        with pytest.warns(DeprecationWarning) as records:
            module.statement()
            assert len(records) == 1
```
`warnings` is also the name of the stdlib module, so it might be better to use something different instead.